lib/core/src/display/navigation/navigator.rs

```diff
@@ -62,9 +66,8 @@ impl Navigator {
         let drag               = DragProperties::new(1500.0);
         let properties         = PhysicsProperties::new(kinematics, spring, drag);
         let steps_per_second   = 60.0;
-        let properties_clone   = properties.clone();
         let callback           = move |position| camera.set_position(position);
-        let sim = PhysicsSimulator::new(steps_per_second,properties_clone,callback);
+        let sim = PhysicsSimulator::new(steps_per_second,properties.clone(),callback);
```

Comment:
I know this is old code, so no need to fix that, ubut this constructor should accept reference like `PhysicsSimulator::new(steps_per_second,&properties),callback);` and do the clone inside if needed. In fadct, it shouldaccept anythign `<S:Into<Properties>>` (so it sohuld accept both reference as well as value). Im wiriting aobut it because when designing APIs lets do interfaces this way :)